### PR TITLE
AVM 8

### PR DESCRIPTION
**Pull request by michaeldiamant**
Provides an integration branch for PRs targeting AVM 8.

Relates to https://github.com/algorand/pyteal/issues/388.

### CODE REVIEW

**Comment by michaeldiamant** [comment]
I can't approve my own PR, so I leave a comment instead:  As of https://github.com/algorand/pyteal/pull/439/commits/5282726f675b65498b04fe9f608694d94414b57e, the integration branch looks ready to merge once AVM 8 release kicks off.

**Review by jasonpaulos — COMMENTED** [review]
This looks pretty good, I just made a child PR for some docs improvements: #546

**Review by ahangsu — COMMENTED** [review]
Mostly look good, just one question on `acct_params_get` op code, and the related fields. Do we want them included in AVM8 release?

**Comment by michaeldiamant** [comment]
> just one question on acct_params_get op code, and the related fields. Do we want them included in AVM8 release?

@ahangsu Thanks for flagging it to ensure we cover all the bases.  

* We're working on it, but I do _not_ view it as a blocker to merging the PR.  This PR brings a lot of value as is.
* If the work is done before releasing AVM 8, then we'll happily take the change.  Otherwise, we'll include it in a subsequent PyTeal release.

**Review by ahangsu — APPROVED** [review]
Feels good about the PR in current shape, approving for I feel like this looks good.

**Review by jasonpaulos — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Unreleased
```
Comment (ahangsu) [inline CHANGELOG.md:1]:
```suggestion
# Unreleased

# 0.20.0

```

**Pull request merged**